playbooks/roles/nginx/tasks/main.yml

```diff
@@ -62,21 +66,41 @@
   notify: reload nginx
   tags: instance-config
 
-- name: Create the Nginx configuration files
+- name: Create the Nginx live configuration files
+  template: src={{ item }}.j2
+            dest=/etc/nginx/sites-available/{{ item }}
+            backup=yes
+  notify: reload nginx
+  with_items: "{{ live_config }}"
+  tags: instance-config
+  when: nginx_config == "live"
+
+- name: Create the Nginx maintenance configuration files
   template: src={{ item }}.j2
             dest=/etc/nginx/sites-available/{{ item }}
             backup=yes
   notify: reload nginx
-  with_items: nginx_configs
+  with_items: "{{ maintenance_config }}"
```

Comment:
`{{ nginx_configs[current_config] }}` and then we might be able to get rid of the duplicate task